apps/platform/components/projects/EnvironmentVariableEditor.tsx

```diff
@@ -189,40 +196,9 @@ export function EnvironmentVariableEditor({ branchId }: { branchId: string }) {
 
                 <div className="col-span-9">
                   <div className="flex items-center gap-3">
-                    <Controller
-                      control={control}
-                      name={`secrets.${index}.decryptedValue` as const}
-                      render={({ field: { onChange, value, name } }) => (
-                        <TextareaGroup
-                          full
-                          icon={<Eye className="text-lighter h-4 w-4" />}
-                          autoComplete="off"
-                          className={clsx(
-                            // envPair.hidden ? "obscure" : "",
-                            "inline-block font-mono",
-                          )}
-                          disabled={false}
-                          iconActionClick={() =>
-                            handleToggleHiddenEnvPairClick(index)
-                          }
-                          name={name}
-                          value={value}
-                          onChange={async (e) => {
-                            onChange(e.target.value);
-
-                            const encryptedSecretValue = await encryptString(
-                              e.target.value,
-                              decryptedProjectKey,
-                            );
-
-                            setValue(
-                              `secrets.${index}.encryptedValue`,
-                              encryptedSecretValue,
-                            );
-                          }}
-                        />
-                      )}
-                    />
+                    <ConditionalInput
```

Comment:
This may have happened during merge conflict. But this `ConditionalInput`, there is syntax error with this one.